src/components/footer/index.md.njk

```diff
@@ -45,6 +45,19 @@ You can add links to:
 
 Use ‘Privacy’, ‘Accessibility’, ‘Cookies’ and ‘Terms and conditions’ for the link text.
 
+## Use localisation to translate ‘footer’ into other languages
+
+Read how to [set up localisation with Nunjucks and Javascript](/get-started/localisation/).
+
+### Add Nunjucks parameters
+
+Add the following parameters to the footer:
+
+- `contentLicence` – the content licence information, this defaults to Open Government Licence (OGL) v3 licence
+- `hideOpenGovernmentLicenceLogo` – hides the OGL logo and is used if the content licence is not the Open Government Licence
+- `copyright` – the copyright information, this defaults to Crown Copyright
+- `hideCrownCopyrightArms` – hides the royal coat of arms and is used if the document is not covered under Crown Copyright
```

Comment:
Not something for now, but just a reminder that this may need changing depending on the change we actually decide to make for the release.